README.pod

```diff
@@ -63,6 +63,27 @@ a subset of the functionality).
 It is based on Martin Berends' MiniDBI project, but unlike MiniDBI, DBDish
 aims to provide an interface that takes advantage of Perl 6 idioms
 
+=head2 Connecting to, and disconnecting from, a database
+
+For a database named I<hierarchy>, a connection may be established with any of
+the following syntaxes:
+
+    my ($dbh, $db);
+
+    $dbh = DBIish.connect("Pg", :database<hierarchy>);
+    $dbh.dispose;
+
+    $db = 'hierarchy';
+
+    $dbh = DBIish.connect("Pg", :database($db));
+    $dbh.dispose;
+
+    $dbh = DBIish.connect("Pg", database => $db);
+    $dbh.dispose;
+
+    $dbh = DBIish.connect("Pg", :$db);
```

Comment:
Well, it worked for me!
```sh
$ perl6 -Ilib -MDBIish -e 'my $db = q|hierarchy|; my $dbh = DBIish.connect("Pg", :$db); say "Database $db: connection ", ($dbh.ping ?? "established" !! "not established"); $dbh.dispose'
Database hierarchy: connection established
```
... where I do indeed have a local Pg db named _hierarchy_.  Do you think that that would not generally work?